vendor/blockchain_contracts/src/bin/calculate_offsets/ethereum/rfc003/templates/out/deploy_header.asm.hex

```diff
@@ -0,0 +1 @@
+6120026110016000396120026000f3
```

Comment:
:question: Why is this being moved here?

Reply:
Switching too much between branches ...